Makefile

```diff
@@ -113,6 +113,13 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 fmt: ## Run go fmt against code.
 	go fmt ./...
 
+.PHONY: fmt-check
+fmt-check:
+	@files=$$(find . -name "*.go" -not -path "./vendor/*" -not -path "./dpu-api/vendor/*"); \
+	output=$$(gofmt -d $$files); \
```

Comment:
Typo? Shouldn't there be a space between go and fmt

Reply:
no, gofmt is capable to do a dry run. `go fmt` isn't.

Reply:
Ah I see. you want to be able to dry run to see what go fmt will do to the code.